lldb/test/API/tools/lldb-dap/server/Makefile

```diff
@@ -0,0 +1,3 @@
+C_SOURCES := main.c
+
+include Makefile.rules
```

Comment:
I think this stop sign indicates a missing newline.

Reply:
Done.